File: 58-length-of-last-word/length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        
        s = s.trim();
        int lastIndex = s.length()-1;
        int count = 0;
        while(lastIndex>=0 && s.charAt(lastIndex)!=' ') {
            count++;
            lastIndex--;
        }
        return count;
    }
}

File: 58-length-of-last-word/length-of-last-word.js
/**
 * @param {string} s
 * @return {number}
 */
var lengthOfLastWord = function(s) {
    let arr = s.trim().split(' ');
    if(arr.length<=1){
        return arr[0].length;
    }
    for(let i = 0;i<arr.length;i++){
        if(i=arr.length-1){
             return arr[i].length
        }
    }
};